### PR TITLE
[16.0][ADD] project_stage_extra_info

### PR DESCRIPTION
**Pull request by dsolanki-initos**
After the discussion with @pedrobaeza on [ project_status PR](https://github.com/OCA/project/pull/1109) I've created this PR
This module adds a Description and Is closed stage field on Project stages.

### CODE REVIEW

**Review by angelmoya — APPROVED** [review]
Code review LGTM

Functional review on runboat is OK

Would be nice to create migration script on project_status, to migrate data to standard project status with this module data.

**Comment by angelmoya** [comment]
README.rst is right? must be generated with mantainer-tools/gen_addons_readme.py?

File: project_stage_extra_info/models/project_status.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+from odoo import fields, models
```
Comment (SkiBY) [inline project_stage_extra_info/models/project_status.py]: Could you please rename file and class names, to be more consistent with stage model

**Review by SkiBY — CHANGES_REQUESTED** [review]

**Comment by SkiBY** [comment]
@angelmoya yes, the script for upgrade could be great, but there is no place to it, no? 16.0 have no status addon at all, so...
Should be the part of OpenUprade probably?

**Comment by dsolanki-initos** [comment]
> README.rst is right? must be generated with mantainer-tools/gen_addons_readme.py?

Hi @angelmoya, Added the generated README.rst

File: project_stage_extra_info/models/project_status.py (hunk 1 above)
Comment (dsolanki-initos) [reply project_stage_extra_info/models/project_status.py]: @SkiBY, I've updated the changes, Kindly check it.

**Comment by angelmoya** [comment]
@SkiBY to do migration script whe need this module, without code, but with migration script, you can check

project_task_dependency on version 15.0

https://github.com/OCA/project/tree/15.0/project_task_dependency

**Comment by SkiBY** [comment]
> @SkiBY to do migration script whe need this module, without code, but with migration script, you can check
> 
> project_task_dependency on version 15.0
> 
> https://github.com/OCA/project/tree/15.0/project_task_dependency

good idea, thank you, but i don't think, this should stop this pr to be merged.

**Review by SkiBY — APPROVED** [review]

**Comment by angelmoya** [comment]
Maybe you can add on roadmap

**Review by leemannd — COMMENTED** [review]
Quick review, base technical work looks good. Won't it need a migration script as we are moving data from 15.0  `project_status` to 16.0 `project_stage_extra_info`
?

**Comment by dreispt** [comment]
@leemannd If the fields are the same, changing the module adding them has no impact.
If this is OK, is the PR good to merge?

**Review by leemannd — APPROVED** [review]

**Pull request merged**